models/compound/ethereum/compound_ethereum_schema.yml

```diff
@@ -97,4 +97,125 @@ models:
               values: ['Queued', 'Active', 'Executed', 'Canceled','Defeated']
       - &description
         name: description
-        description: "Description of the proposal"
\ No newline at end of file
+        description: "Description of the proposal"
+
+  - name: compound_v2_ethereum_ctokens
+    meta:
+      blockchain: ethereum
+      sector: lending
+      project: compound_v2
+      contributors: bizzyvinci
+    config:
+      tags: ['ethereum', 'compound', 'compound_v2', 'cToken', 'cEther', 'cErc20']
+    description: "All compound v2 cTokens"
+    columns:
+      - name: asset_symbol
+        description: "Underlying asset symbol"
+      - name: ctoken_address
+        description: "cToken address"
+      - name: asset_address
+        description: "Underlying asset address"
+      - name: decimals_mantissa
+        description: "10 raise to power decimals of underlying asset"
+
+  - name: compound_v2_ethereum_supply
+    meta:
+      blockchain: ethereum
+      sector: lending
+      project: compound_v2
+      contributors: [bizzyvinci, hosuke]
+    config:
+      tags: ['ethereum', 'compound', 'compound_v2', 'supply', 'lending']
+    description: "All compound v2 transactions involving supply transactions on ethereum"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - version
+            - transaction_type
+            - token_address
+            - evt_tx_hash
+            - depositor
+            - withdrawn_to
```

Comment:
Also can you help me to define the `unique key` combination for `compound_v2_ethereum_supply`?
We can add uniqueness tests in the schema file.